x/supplier/client/cli/query_claim.go

```diff
@@ -52,24 +81,35 @@ func CmdListClaim() *cobra.Command {
 
 func CmdShowClaim() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "show-claim <index>",
-		Short: "shows a claim",
-		Args:  cobra.ExactArgs(1),
+		Use:   "show-claim <session_id> <supplier_addr>",
+		Short: "shows a specific claim",
+		Long: `List a specific claim that the node being queried has access to (if it still exists)
```

Comment:
Do claims get deleted over time?

Reply:
Yea, the natural claim & proof life cycle will delete either proven or expired claims.